edx_credentials_themes/static/edx.org/certificates/sass/_utilities.scss

```diff
@@ -0,0 +1,113 @@
+$em-base: 16; // deliberately sets bourbon-based em-base (http://bourbon.io/docs/#em-base)
+$dark-gray: rgb(17, 16, 16);
+
+$micromasters-color: #065784;
+$micromasters-watermark-color: #005585;
+$xseries-color: #424242;
+$professional-certificate-color: #9a2061;
+$professional-certificate-watermark-color: $xseries-color;
+
+$edx-e-color: #b72768;
+$edx-e-d-overlap-color: #78212e;
+$edx-d-color: #8a8c8f;
+$edx-d-x-overlap-color: #026ba4;
+$edx-x-color: #209fda;
+
+// #bootstrapupgrade
+// Colors from edX bootstrap. Remove these when we update the libraries.
```

Comment:
A TODO might be helpful.

Do we have a ticket open so we don't forget to do this?